src/utils/proxyMap.ts

```diff
@@ -0,0 +1,113 @@
+import { proxy } from 'valtio'
+
+type KeyValRecord<K, V> = {
+  key: K
+  value: V
+}
```

Comment:
I was thinking of this:
```js
type KeyValueRecord<K, V> = [key: K, value: V]
```
but either is okay.

actually, this might generate smallest bundle:
```js
type KeyValRecord<K, V> = {
  k: K
  v: V
}
```

Reply:
You decided to keep it as is, right? Just confirming as this is marked as resolved. Let me unmark it.